### PR TITLE
chore: remove preserve-paths package

### PR DESCRIPTION
**Pull request by lazysoundsystem**
Refs: OPS-9348

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -83,14 +82,14 @@
...
-            "drupal-composer/preserve-paths": true,
             "drupal/console-extend-plugin": true,
             "drupal/core-composer-scaffold": true,
             "drupal/core-project-message": true,
             "oomphinc/composer-installers-extender": true,
             "orakili/composer-drupal-info-file-patch-helper": true,
             "phpstan/extension-installer": true,
-            "symfony/flex": true
+            "symfony/flex": true,
+            "drupal-composer/preserve-paths": true
```
Comment (orakili) [inline composer.json:92]: Why is this needed if the plugin is being removed?

Comment (lazysoundsystem) [reply composer.json]: Never worked out why composer wanted to put that line back in. I've removed it again and it seems okay with that now.

**Review by orakili — APPROVED** [review]
